### PR TITLE
New data

### PR DESCRIPTION
**Pull request by cwehmeyer**
Added a generator for multi-ensemble data.

### CODE REVIEW

File: doc/source/CHANGELOG.rst [hunk 1]
```diff
@@ -28,6 +28,9 @@ Changelog
    - kmeans: allow the random seed used for initializing the centers to be passed. The prior behaviour
      was to init the generator by time, if fixed_seed=False. Now bool and int can be passed. #1091
 
+- datasets:
+   - added a multi-ensemble data generator for the 1D asymmetric double well.
```
Comment (marscher) [inline doc/source/CHANGELOG.rst]: please add the number of this PR here.

Comment (cwehmeyer) [reply doc/source/CHANGELOG.rst]: done

**Review by marscher — CHANGES_REQUESTED** [review]
Could you please add a simple invocation of this, to see it does not fail? Actually I trust you that you've tested this well, but if somebody is going to change (break) it in the future, we have no chance to catch this early.

Put it in datasets.tests.test_datasets.

**Review by marscher — APPROVED** [review]
perfect!

**Comment by marscher** [comment]
I just cancelled the second appveyor job (there is no need to test both the branch and the PR merge to devel in this case).

**Pull request merged**